onnxruntime/core/providers/migraphx/migraphx_execution_provider.h

```diff
@@ -54,9 +54,10 @@ class MIGraphXExecutionProvider : public IExecutionProvider {
 
   void RegisterAllocator(AllocatorManager& allocator_manager) override;
 
-  void* GetComputeStream() const override { return static_cast<void*>(stream_); }
+  void* GetComputeStream() const { return static_cast<void*>(stream_); }  // TODO: this function should be deleted
```

Comment:
why you still keep it? this API should be removed.

Reply:
No it shouldn't MIGraphX requires this to operate.